src/reducers/index.js

```diff
@@ -1499,7 +1502,6 @@ export const onUpdateNetworkSettings = (state, { payload }) => {
     networkSettings: {
       ...state.networkSettings,
       data: payload,
-      status: NETWORK_SETTINGS_STATUS.SUCCESS,
```

Comment:
Remove this because we need to change the status to success only after all methods are executed (added some comments explaining this in the saga)